cioos_yaml_to_erddap/yaml_to_erddap_snippet.py

```diff
@@ -25,6 +25,18 @@ def yaml_to_erddap_dict(record: Dict) -> Dict:
     if not language:
         raise Exception("'language' cannot be empty")
 
+    # Define Info URL and id
+    infoUrl = None
+    if record['identification'].get("identifier"):
```

Comment:
You might want to add a check to see if identifier is already a url. the records coming from the form won't be, but a few people will use this package on yaml files

Reply:
or could check to see if `identifier` is a DOI

Reply:
Ok, I wasn't sure it looked like the `identifier` field within the yaml from the cioos metadata form was coming from the doi field in the form. Didn't know it could have other identifier sources.